src/js/modules/supervisors/FileManager.ts

```diff
@@ -256,6 +266,9 @@ class FileManager {
     if (coprocessors.length == 0) {
       return Promise.resolve();
     } else {
+      logger.info(
```

Comment:
the line has 80 characters for line

Reply:
I ran the formatter though, `vtools fmt js` is there another way ?

Reply:
I remember `prettier` doesn't split string lines, you make it by your-self, you can run `npm run prettier:check` in order to verify that